openpype/plugins/publish/help/validate_containers.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<root>
+<error id="main">
+<title>Not up-to-date assets</title>
+<description>
+## Obsolete containers found
+
+Scene contains one or more obsolete loaded containers, eg. items loaded into scene by Loader.
```

Comment:
I wonder whether "obsolete" is the right word to use here? It sounds like 'don't use the subset anymore please' instead of being "outdated" or "old". Thoughts?